assets/src/components/propertiesPanel/blockWaiverBanner.tsx

```diff
@@ -0,0 +1,128 @@
+import React from "react"
+import { alertCircleIcon } from "../../helpers/icon"
+import { BlockWaiver } from "../../realtime"
+
+interface Props {
+  blockWaiver: BlockWaiver
+}
+
+export const hours12 = (hours24plus: number): number => {
```

Comment:
Since this is now using the output of `date.getHours()` instead of the GTFS time of day, I don't think this can be bigger than 23 anymore. (Same for `ampm()`)

It'll also need a 0 case now.

Reply:
Updated